### DISCUSSION TITLE
Average rain gauge observations within a forecast grid cell

### DISCUSSION THREAD

**Discussion by rvalenzuelar** [dtcenter/METplus#980] [Use Cases] [answered]
I would like to compute the average precipitation of all observation points inside a given forecast grid cell (e.g. verify average rain gauge precipitation and forecast value within the same grid cell). I went through all the PDFs in the tutorial but I am not sure how to achieve this. I though it was with Series-Analysis but the PDF only shows a grid-grid example.

**Comment by jprestop** [comment]
Thank you for submitting your question.  I think @JohnHalleyGotway is the best person to help you with your question.  Please allow a few business days for a response.

**Comment by JohnHalleyGotway** [comment, marked as answer]
I see you have a question about computing the average value across multiple stations falling within the same grid cell.

In general, when verifying gridded NWP forecast data against point observations, you'd run the MET Point-Stat tool. However, Point-Stat processes each observation station individually. There is no existing mechanism by which it can recognize that multiple stations fall within the same grid box and compute an average of their values. Although, if a single station reports multiple times within a time window, you can configure Point-Stat to compute a summary over those multiple reports. That's handled by the "[obs_summary](https://met.readthedocs.io/en/develop/Users_Guide/config_options.html#obs-summary)" configuration option. And typically, this is more common than having multiple stations falling within the same grid box. So Point-Stat doesn't do what you want.

However, the Point2Grid tool might. This tool reads point observations and write gridded fields of data. It essentially "grids" the point data. So if your point data is dense enough to include multiple reports within the same grid box, this may be a good application of that tool. And in that case, yes, you can specify that the value for the grid box be computed as the mean value of all points falling inside of it. Once you've generated a gridded analysis field using Point2Grid, you can then run MET's Grid-Stat tool to compare your gridded forecast to gridded analysis.

Hope that helps clarify the options.

**Reply by rvalenzuelar** [reply]
Thanks John for the detailed answer. The workflow you mention makes sense to me. I will report the results when I have it. Best!

**Reply by rvalenzuelar** [reply]
Hi, just reporting that it worked as you indicated. Thanks!
